packages/devtools_app/lib/src/debugger/flutter/debugger_controller.dart

```diff
@@ -836,3 +856,74 @@ class ScriptCache {
     _inProgress.clear();
   }
 }
+
+/// Maintains the navigation history of the debugger's code area - which files
+/// were opened, whether it's possible to navigate forwards and backwards in the
+/// history, ...
+class ScriptsHistory extends ChangeNotifier
+    implements ValueListenable<ScriptsHistory> {
+  // TODO(devoncarew): This class should also record and restore scroll
+  // positions.
+
+  ScriptsHistory();
+
+  final List<ScriptRef> _history = [];
+  int _historyIndex = -1;
+
+  final Set<ScriptRef> _openedScripts = {};
```

Comment:
rewrite as `final _openedScripts = <ScriptRef>{};`

same for history above

Reply:
done